escrow/lock.go

```diff
@@ -19,10 +19,15 @@ type Locker interface {
 // NewEtcdLocker returns new lock which is based on etcd storage.
 func NewEtcdLocker(storage AtomicStorage) Locker {
 	return &etcdLocker{
-		storage: &PrefixedAtomicStorage{
-			delegate:  storage,
-			keyPrefix: "/payment-channel/lock",
-		},
+		storage: NewPrefixedStorage(storage),
+	}
+}
+
+// returns new prefixed storage
+func NewPrefixedStorage(storage AtomicStorage) *PrefixedAtomicStorage {
```

Comment:
Same thing as in https://github.com/singnet/snet-daemon/pull/187/files#r248655810 Let's rename it to something less abstract for instance `NewEtcdLockerStorage` or `NewLockerStorage`. `PrefixedAtomicStorage` type is used few times in code for different storages.

Reply:
LockerStorage would be ideal i think. And changed the code accordingly.